ajax/libs/dropbox.js/package.json

```diff
@@ -1,29 +1,35 @@
 {
   "name": "dropbox.js",
-  "filename": "dropbox.min.js",
-  "version": "0.10.3",
-  "description": "Client library for the Dropbox API",
+  "filename": "Dropbox-sdk.min.js",
+  "version": "2.1.0",
+  "description": "The Dropbox JavaScript SDK is a lightweight, promise based interface to the Dropbox v2 API that works in both nodejs and browser environments.",
```

Comment:
We also have non-v2 versions, and we didn't host the node version, maybe not to touch desc is better.